addons/stock/models/stock_warehouse.py

```diff
@@ -346,6 +346,9 @@ def _create_or_update_sequences_and_picking_types(self):
             else:
                 data[picking_type].update(create_data[picking_type])
                 sequence = IrSequenceSudo.create(sequence_data[picking_type])
+                existing_sequence = IrSequenceSudo.search([('company_id', '=', sequence.company_id.id), ('name', '=', sequence.name)])
```

Comment:
```suggestion
                existing_sequence = IrSequenceSudo.search_count([('company_id', '=', sequence.company_id.id), ('name', '=', sequence.name)], limit=1)
```
You don't need to get the sequence values. just to know if there is one

Reply:
thanks for the review. done